lib/web_ui/dev/chrome.dart

```diff
@@ -157,7 +150,27 @@ Future<Process> _spawnChromiumProcess(String executable, List<String> args, { St
     // launching more processes.
     process.exitCode.timeout(const Duration(seconds: 1), onTimeout: () {
       process.kill();
-      return null;
+      return -1;
     });
   }
 }
+
+/// Returns the full URL of the Chrome remote debugger for the main page.
+///
+/// This takes the [base] remote debugger URL (which points to a browser-wide
+/// page) and uses its JSON API to find the resolved URL for debugging the host
+/// page.
+Future<Uri> getRemoteDebuggerUrl(Uri base) async {
```

Comment:
This is not nullability related. Just removing a dependency on an implementation detail (`package:test_core/src`).